src/Base/Console.h

```diff
@@ -470,7 +470,7 @@ class BaseExport ConsoleObserver
     virtual void Log    (const char *){}
 
     virtual const char *Name(void){return 0L;}
-    bool bErr,bMsg,bLog,bWrn;
+    bool bErr,bMsg,bLog,bWrn,bShow;

```

Comment:
There is not need to add a new member to the very base class which is only used by a sub-class. But not even in the sub-class it's really needed. See the comments there...